envoy/auth_server/main.go

```diff
@@ -97,12 +111,23 @@ func main() {
 		panic(err)
 	}
 
+	// Determine which gateway endpoint ID extractor to use
```

Comment:
Thoughts on creating a small helper for this?

Reply:
Done.